charts/dgraph/values.yaml

```diff
@@ -10,7 +10,7 @@
 image: &image
   registry: docker.io
   repository: dgraph/dgraph
-  tag: v21.12.0
+  tag: v22.0.2
```

Comment:
DRY: I think we can reference the `appVersion` as the image `tag` for `dgraph`, this will need a template change to facilitate this. 

We may accidentally miss changing this in multiple places.

Reply:
Code snippet?  That's a good idea actually.  This wasn't done in the past, because they started using super hero names as docker image tags, which were inconsistent to dgraph versions.  But this shouldn't be a problem now.

Reply:
@darkn3rd I didn't realize before, but you cannot change helm chart config values, as helm does not support dynamic config values.  These are set by the user (operator) to that replace parts in the helm chart manifests.  Thus this request is not valid, because it is not possible with helm technology.